### PR TITLE
OOP

### PR DESCRIPTION
**Pull request by danacebryantalfonso**

### CODE REVIEW

**Comment by danacebryantalfonso** [comment]
Changed some files in 696b5d54cb5637e148c390201ac0061ffb7dcf39

File: OOP/src/CompanionObjectPractice.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import kotlin.random.Random
+
+fun main(args: Array<String>) {
+    val correctKey :Key = Lock.createKey()
```
Comment (rexendy-lig) [inline OOP/src/CompanionObjectPractice.kt:4]: val correctKey: Key = Lock.createKey()

File: OOP/src/Constructors.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    yourCar.run()
+    myCar.run()
+}
+
+class CarConstructor {
+    constructor() {
+        model = "No model"
+        topSpeed = 150
+    }
+    constructor(newModel: String) {
```
Comment (rexendy-lig) [inline OOP/src/Constructors.kt:16]: pls put next line

File: OOP/src/Exercise3.kt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    var balance = 100
+    val price = 20
+
+    fun run(){
+        var qty = balance/price
+        if(balance >= price) {
+            println("$userName can buy $qty pcs")
+        }else{
+            println("$userName can't afford to buy.")
+        }
```
Comment (rexendy-lig) [inline OOP/src/Exercise3.kt]: pls apply proper spacing or

if (balance >= price) 
            println("$userName can buy $qty pcs")
  else
            println("$userName can't afford to buy.")
       

you can use {} if you have a lot of function to do inside {}

**Review by rexendy-lig — CHANGES_REQUESTED** [review]

File: OOP/src/Exercise3.kt (hunk 3 above)
Comment (danacebryantalfonso) [reply OOP/src/Exercise3.kt]: fixed on this 696b5d54cb5637e148c390201ac0061ffb7dcf39

File: OOP/src/CompanionObjectPractice.kt (hunk 1 above)
Comment (danacebryantalfonso) [reply OOP/src/CompanionObjectPractice.kt]: done in 2dd24c712eae9dd0773ca38ab20ba74ee029b02f

File: OOP/src/Constructors.kt (hunk 2 above)
Comment (danacebryantalfonso) [reply OOP/src/Constructors.kt]: done in 2dd24c712eae9dd0773ca38ab20ba74ee029b02f

**Review by rexendy-lig — APPROVED** [review]

**Pull request merged**